GetDKAN/dkan
# Added conditional to prevent search_api from attempting to index an unpublished dataset.

## QA Steps
At this url:
admin/config/workflow/workflows/manage/dkan_publishing
Change default moderation state to be 'Draft'

register and run a harvest. The result of this added code is that this warning message should not be printed out:
`[warning] Could not load the following items on index DKAN: ...`
